submissions/MarharytaBoichenko/oop_frogger/app.js

```diff
@@ -0,0 +1,110 @@
+// Enemies our player must avoid
+const FIELDWIDTH = 505;
+const FIELDHEIGHT = 606;
+const playerStartX = 203;
+const playerStartY = 300;
+const playerStepY = 83;
+const playerStepX = 101;
```

Comment:
I feel there is a connection between 505, 606, 101, 203... Can we make some of the constants calculated from others?